lib/plugins/aws/package/compile/events/rabbitmq.js

```diff
@@ -101,6 +104,21 @@ class AwsCompileRabbitMQEvents {
         const lambdaLogicalId = this.provider.naming.getLambdaLogicalId(functionName);
         const dependsOn = this.provider.resolveFunctionIamRoleResourceName(functionObj) || [];
 
+        const sourceAccessConfigurations = [
+          {
+            Type: 'BASIC_AUTH',
+            URI: basicAuthArn,
+          }
+        ]
+
+        // if not specified it will default to /
```

Comment:
It's not clear what is the source of that default - is that the default behavior on AWS side? If that's the case, I think we can safely remove this comment from the code

Reply:
it defaults it to a / in AWS will remove the comment